### ISSUE TITLE
Parsing not understand type MUST not return error

### ISSUE THREAD

**Issue by teeson** [gortc/sdp#8]
An SDP example taken from the README file 

> ```
> v=0
> m=video 51372 RTP/AVP 99
> a=rtpmap:99 h263-1998/90000
> e=test@test.com
> ü=vαlue
> ```

when we use Decoder to Decode we got an error

` "failed to decode media"`

```
var (
    s sdp.Session
)

s, err = sdp.DecodeSession(session, s)
d := sdp.NewDecoder(s)
media := new(sdp.Message)
if err = d.Decode(media); err != nil {
	log.Fatal("err:", err)
}
```

AS RFC4566 spec:

> 5.  SDP Specification
> ...
> The set of type letters is deliberately small and not intended to be
> extensible -- an SDP parser MUST completely ignore any session
> description that contains a type letter that it does not understand.
> ...

So I advise that the Decoder should save unknown type description in a generic struct field
, or simply discard that description line instead of return error

**Comment by ernado** [comment]
Thank you! I will fix it shortly.

**Comment by ernado** [comment]
Should be fixed in v0.12.5

**Comment by teeson** [comment]
Thanks for your quick response
